zerver/tests/test_home.py

```diff
@@ -704,11 +704,11 @@ def test_invites_by_admins_only(self) -> None:
 
         self.login_user(user_profile)
         self.assertFalse(user_profile.is_realm_admin)
-        self.assertNotInHomePage("Invite more users")
```

Comment:
`assertNotInHomePage` and `assertInHomePage` basically checks for a string that is either absent or present in the `index.html` template. However, since the `right_sidebar` is migrated to handlebars, `assertInHomePage` will always return `false`.
The way I modified the tests are so that they `assertEqual` if the user_profile invite policy matches the `page_params.show_invites` (Because this is what we check within handlebars). 

Any suggestions regarding this would help :)

Reply:
I think the purpose of this function is to check if a user can invite others, which is already tested by `test_compute_show_invites_and_add_streams*` functions

Reply:
Yeah, agreed, I think we can just delete this entire test, perhaps in a commit message explaining that the behavior is already tested by the other logic for deciding what to pass in `page_params`.

(We may post-migration want to add new tests in the Handlebars world)

Reply:
Yup, Thanks @ganpa3 @timabbott!